packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -12,9 +12,13 @@ import 'package:ubuntu_utils/ubuntu_utils.dart';
 import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru/yaru.dart';
 
+const defaultFilePath = '/usr/share/desktop-provision';
+
 final eulaPageProvider = FutureProvider<File>((ref) async {
   final locale = Intl.defaultLocale ?? await findSystemLocale();
-  const directory = '/usr/share/desktop-provision/eula';
+  final directory = p.join(
+      Platform.environment['DESKTOP_PROVISION_PATH'] ?? defaultFilePath,
+      'eula');
```

Comment:
```suggestion
      'eula',);
```
Add trailing comma, needs formatting afterwards